.github/workflows/test-integration.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         run: |
           nix build -L .#wasm
           mkdir -p ./.packages/noir_wasm
-          cp -r ./result/* ./.packages/noir_wasm/
+          cp -r ./result/noir_wasm/ ./.packages/noir_wasm/
```

Comment:
@kobyhallx we should discuss these changes (the build-all-by-default nix change)

Reply:
If we have nargo as default, then we could utilize .#all to get all at the same time